pkg/controller/stats/aggregator.go

```diff
@@ -369,3 +381,40 @@ func addUp(stats *statsv1alpha1.TrafficStats, inc *statsv1alpha1.TrafficStats) {
 	stats.Packets += inc.Packets
 	stats.Bytes += inc.Bytes
 }
+
+func addRulesUp(ruleStats *[]statsv1alpha1.RuleTrafficStats, ruleSumStats *statsv1alpha1.TrafficStats, inc []statsv1alpha1.RuleTrafficStats) {
+	incMap := make(map[string]statsv1alpha1.TrafficStats)
+	for _, v := range inc {
+		incMap[v.Name] = statsv1alpha1.TrafficStats{
```

Comment:
Instead of making a new TrafficStats, the map value's type can be `*statsv1alpha1.RuleTrafficStats` to refer to the original value directly.